integration/setup/setup.go

```diff
@@ -142,7 +155,7 @@ func SetupWithOpts(tb testtb.TB, opts *SetupOpts) *SetupResult {
 
 	uri := *targetURLF
 	if uri == "" {
-		uri = setupListener(tb, setupCtx, logger)
+		uri = setupListener(tb, setupCtx, logger, opts.BackendOptions)
```

Comment:
I think we should set default value of opts.BackendOptions in this file, because this is where `BackendOptions` is defined :hugs:

Reply:
Added `NewBackendOpts()` to handle default values.